### PR TITLE
ditch ftp post

### PR DESCRIPTION
**Pull request by njrifkind**
Blog post on reasons to ditch FTP.

### CODE REVIEW

File: _posts/2015-11-05-ftp.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+Photo credit: Sunset Boulevard/Corbis, [Time.com](http://time.com/59475/star-wars-episode-seven-morocco-tunisia/).
+
+Since we're going to assume your company is without it's own quirky yet competent astromech droid, odds are then that you need to take data security a bit more seriously. And yes, there are steps you can take to mitigate [FTP's inherent security gaps and flaws](http://www.southrivertech.com/five-ftp-server-security-fails-ftp-issues-and-how-to-avoid-them/). That list includes everything from straight-up unencrypted data to allowing anonymous access. It's hard to read through it and come away thinking FTP still is a viable choice for your business - especially so in a world where [managed file transfer](https://en.wikipedia.org/wiki/Managed_file_transfer) (MFT) exists. 
+
+## FTP clients are hard to use
+
+Compared to modern web services, FTP clients are a pain to install, configure, and use...but at least they also offer fewer security, auditing and monitoring capabilities. Many of the factors that made FTP the go-to file transfer option are [simply no longer relevant](https://mxforum.mendix.com/questions/4892/FTP-vs-Webservices).
```
Comment (mikepmunroe) [inline _posts/2015-11-05-ftp.md]: I'm not sure I follow what is better here. I think I follow the point you are trying to make, but the wording makes it hard to follow which is proposed to be the better option.

File: _posts/2015-11-05-ftp.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+![FTP is insecure]({{ BASE_PATH }}/assets/images/ftp/FTP-is-insecure.jpg)
+Photo credit: Sunset Boulevard/Corbis, [Time.com](http://time.com/59475/star-wars-episode-seven-morocco-tunisia/).
+
+Since we're going to assume your company is without it's own quirky yet competent astromech droid, odds are then that you need to take data security a bit more seriously. And yes, there are steps you can take to mitigate [FTP's inherent security gaps and flaws](http://www.southrivertech.com/five-ftp-server-security-fails-ftp-issues-and-how-to-avoid-them/). That list includes everything from straight-up unencrypted data to allowing anonymous access. It's hard to read through it and come away thinking FTP still is a viable choice for your business - especially so in a world where [managed file transfer](https://en.wikipedia.org/wiki/Managed_file_transfer) (MFT) exists.
```
Comment (mikepmunroe) [inline _posts/2015-11-05-ftp.md]: I know MFT is utilized here and a few spots following, but I would replace MFT with a different option. 

`It's hard to read through it and come away thinking FTP still is a viable choice for your business - especially so in a world where [web service] (http://www.drdobbs.com/web-development/restful-web-services-a-tutorial/240169069) communication between servers is so common. Often, we see FTP used to exchange XML files that could easily be replaced with a standard web service`

Comment (njrifkind) [inline _posts/2015-11-05-ftp.md]: Cleaned this and the later reference up by adding another alternative along with MFT, WebDAV, as examples of alternative toolsets/approaches. LMK if there's a specific technology that you'd like me to call out here. (I hit the limits of what my Google-Fu was turning up.)

File: _posts/2015-11-05-ftp.md (hunk 1 above)
Comment (njrifkind) [inline _posts/2015-11-05-ftp.md]: Cleaned this up; good catch.

File: _posts/2015-11-05-ftp.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+![FTP is insecure]({{ BASE_PATH }}/assets/images/ftp/FTP-is-insecure.jpg)
+Photo credit: Sunset Boulevard/Corbis, [Time.com](http://time.com/59475/star-wars-episode-seven-morocco-tunisia/).
+
+Since we're going to assume your company is without it's own quirky yet competent astromech droid, odds are then that you need to take data security a bit more seriously. And yes, there are steps you can take to mitigate [FTP's inherent security gaps and flaws](http://www.southrivertech.com/five-ftp-server-security-fails-ftp-issues-and-how-to-avoid-them/). That list includes everything from straight-up unencrypted data to allowing anonymous access. It's hard to read through it and come away thinking FTP still is a viable choice for your business - especially so when alternative tools ranging from [managed file transfer](https://en.wikipedia.org/wiki/Managed_file_transfer) (MFT) to web distributed authoring and versioning are readily available.
```
Comment (njrifkind) [inline _posts/2015-11-05-ftp.md]: Supplemented the mention of MFT, as per your previous note, by adding in WebDAV as an additional option. LMK if there's a specific technology you'd like called out. (Hit the limit of my technical knowledge trying to ferret out options on this.)

File: _posts/2015-11-05-ftp.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+
+## FTP clients are hard to use
+
+Compared to modern web services, FTP clients are a pain to install, configure, and use. Not only that, they also offer fewer security, auditing and monitoring capabilities. Many of the factors that made FTP the go-to file transfer option are [simply no longer relevant](https://mxforum.mendix.com/questions/4892/FTP-vs-Webservices).
+
+Improved broadband connectivity - in terms of both speed and stability - and new concepts around what constitutes "secure" and efficient data transfer have made web services the easier, cheaper option. And although different web services will solve the problem in different ways, the benefits can be broken down into three main groups:
+
+1. Security: Most modern FTP alternatives approach security in different ways but offer the same or a similar suite of features, including reporting, automation, auditability, encryption, and performance metrics. Gone are the days when it was OK to send usernames and passwords as plain text over FTP. And that's a good thing.
```
Comment (njrifkind) [inline _posts/2015-11-05-ftp.md:39]: Cleaned up the wording. Good catch.

File: _posts/2015-11-05-ftp.md (hunk 3 above)
Comment (mikepmunroe) [inline _posts/2015-11-05-ftp.md]: I would dump MFT in general. I just don't ever see us suggesting MFT as a solution. I would say, pull it out and just be generic. `especially when alternative tools are available`.
